### ISSUE TITLE
copy links to other applications

### ISSUE THREAD

**Issue by derekvan** [FoldingText/foldingtext-for-atom#63]
I'd like to be able to copy a link to a node and paste it into another application on the mac (e.g., DevonThink). I tried dragging/dropping, but that only works within Atom docs.

**Comment by RobTrew** [comment]
Quick comment: drag and drop would certainly be good if the Atom architecture allows it. 

(In the meanwhile **⇧^ C**  or **Atom > Edit Copy Path** gives a link to the selected item which can be used in Devonthink for a `Data > New Bookmark`, which works well and gives a preview)

![screen shot 2015-06-26 at 15 59 24](https://cloud.githubusercontent.com/assets/2305761/8380295/e23baf18-1c1c-11e5-8215-cd4f37e01b33.png)

**Comment by derekvan** [comment]
That's an interesting use case, but it doesn't open the FTML file, then select the specific line / section. That's what I'm really looking for (similar to the ftdoc:// links in FT2).

**Comment by RobTrew** [comment]
I agree, selection would be good – and in fact I think it ought to select already (the url has a selection query appended).  I wonder if there is a bug associated with percent escaping certain characters in the ID string …

**Comment by RobTrew** [comment]
(In DT of course, it's the url on the top edge of the preview that one needs to click (in order to open the file in Atom) - clicking the item name just opens a local DT window.

I'll open a separate issue: creating a special handler like **ft3doc://** or something
